lib/shared/server-request/src/index.ts

```diff
@@ -0,0 +1,136 @@
+import fetchWithRetry, { RequestInitWithRetry } from 'fetch-retry'
```

Comment:
dont love having implementation in `index.ts` files because it makes it hard to keep track of open files in your editor when there's a bunch of them named "index"

Reply:
will rename.